pkg/reconciler/pipelinerun/resources/pipelineref.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2020 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"fmt"
+
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
```

Comment:
Will this still work if the PipelineRun is a v1beta1 PipelineRun object?

Reply:
Right, with https://github.com/tektoncd/pipeline/pull/2413 and https://github.com/tektoncd/pipeline/issues/2410, we will be able to use `v1beta1` everywhere :wink: